packages/dataviews/src/components/dataviews/stories/index.story.tsx

```diff
@@ -208,7 +237,7 @@ export const FreeComposition = () => {
 			paginationInfo={ paginationInfo }
 			data={ processedData }
 			view={ view }
-			fields={ [] }
+			fields={ fields }
 			onChangeView={ setView }
 			defaultLayouts={ defaultLayouts }
 		>
```

Comment:
I still think passing planet as a prop to PlanetOverview is confusing. I know it's possible, but the point of the story is to highlight how to work with DataViews subcomponents, so I'd rather remove this property and access the filtered data via context within the component to highlight how that works.